### PR TITLE
UITEN-257: Tenant settings: Ensure your settings HTML page title follow this format - <<App name>> settings - <<selected page name>> - FOLIO

### PR DESCRIPTION
**Pull request by UladzislauKutarkin**
…ow this format - <<App name>> settings - <<selected page name>> - FOLIO

## Purpose

[UITEN-257](https://issues.folio.org/browse/UITEN-257)
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Comment by UladzislauKutarkin** [comment]
@folio-org/fe-tl-reviewers I have to skip few tests, because I don't know how to make them work, since I cant mock stripes/core in that test cases

File: src/settings/LocationLocations/LocationForm/LocationFormContainer.js [hunk 1]
```diff
@@ -85,12 +86,14 @@ const LocationFormContainer = ({
   }, [onSave, servicePointsByName, saveData]);
 
   return (
-    <LocationForm
-      {...rest}
-      parentMutator={parentMutator}
-      initialValues={initialValues}
-      onSubmit={saveLocation}
-    />
+    <TitleManager page={intl.formatMessage({ id: 'ui-tenant-settings.settings.location.creatNew.title' })}>
```
Comment (zburke) [inline src/settings/LocationLocations/LocationForm/LocationFormContainer.js:89]: Typo: `creatNew` should be `createNew`, or even just `create`.

File: src/settings/Plugins/Plugins.test.js [hunk 2]
```diff
@@ -23,7 +24,7 @@ const ConnectedPlugins = stripesConnect(Plugins, {
 
 const renderPlugins = (props) => renderWithRouter(<ConnectedPlugins {...props} />);
 
-describe('Plugins', () => {
+describe.skip('Plugins', () => {
```
Comment (zburke) [inline src/settings/Plugins/Plugins.test.js:27]: Why can't you mock stripes-core in the test cases? There is an existing mock at `./test/jest/__mocks__/stripesCore.mock.js`; can it be extended?

File: translations/ui-tenant-settings/en.json [hunk 3]
```diff
@@ -1,6 +1,21 @@
...
+  "settings.plugins.title": "Tenant settings - Preferred plugins",
+  "settings.sso.title": "Tenant settings - SSO settings",
+  "settings.service.title": "Tenant settings - Service points",
+  "settings.location.institutions.title": "Tenant settings - Institutions",
+  "settings.location.libraries.title": "Tenant settings - Libraries",
+  "settings.location.campuses.title": "Tenant settings - Campuses",
+  "settings.location.locations.title": "Tenant settings - Locations",
+  "settings.newService.title": "Tenant settings - Create new service point",
+  "settings.items.title": "Tenant settings - {item}",
+  "settings.location.creatNew.title": "Tenant settings - create new location",
```
Comment (zburke) [inline translations/ui-tenant-settings/en.json:16]: `<petty>`
1. As above, change the key to either `createNew` or just `create`. 
2. Capitalize values consistently across entries. Other values use "Tenant settings - Uppercase" so do that here, too.

`</petty>`

**Review by zburke — CHANGES_REQUESTED** [review]
Most changes look just fine, save for a typo in one translation key. 

Let's figure out how to mock stripes-core's Plugin component in order to avoid turning off tests. I'm happy to help if you want to pair on this.

File: src/settings/LocationLocations/LocationForm/LocationFormContainer.js (hunk 1 above)
Comment (UladzislauKutarkin) [reply src/settings/LocationLocations/LocationForm/LocationFormContainer.js:89]: done! Thanks for notions

File: src/settings/Plugins/Plugins.test.js (hunk 2 above)
Comment (UladzislauKutarkin) [reply src/settings/Plugins/Plugins.test.js]: I fixed already that tests

File: translations/ui-tenant-settings/en.json (hunk 3 above)
Comment (UladzislauKutarkin) [reply translations/ui-tenant-settings/en.json:16]: Done!

**Review by zburke — APPROVED** [review]
LGTM

**Pull request merged**